### PR TITLE
Moved formulas and commands into own file

### PR DESCRIPTION
**Pull request by aewens**
In regards to issue [#1](https://github.com/cernodile/Intellect/issues/1)

### CODE REVIEW

**Comment by cernodile** [comment]
Can you resolve the conflict with the current edits? I'll merge once you've done that.

**Comment by aewens** [comment]
Does this work?

File: commands.js [hunk 1]
```diff
@@ -0,0 +1,337 @@
+var formulas = require("./formulas.js");
```
Comment (cernodile) [inline commands.js:1]: You're defining `formulas` here, while you're using `formula` below. That causes the bot to error before even booting up.

**Review by cernodile — CHANGES_REQUESTED** [review]

**Comment by cernodile** [comment]
Ignore the codacy thing, I'm still configuring it.

**Review by cernodile — APPROVED** [review]

File: commands.js (hunk 1 above)
Comment (aewens) [reply commands.js:1]: I fixed the typos

**Comment by cernodile** [comment]
Thank you, hope I didn't make it too hard or easy, and hopefully you learned at least anything today. Have a nice Hacktoberfest!

**Comment by aewens** [comment]
Thanks for the opportunity, happy Hacktoberfest to you as well!

**Pull request merged**